src/components/boostForm.jsx

```diff
@@ -0,0 +1,185 @@
+
+      if (webLnAvailable) {
+        try {
+          await webln.enable();
+          const result = await window.webln.sendPayment(paymentRequest);
+          if (result.preimage) {
+            setIsLoading(false);
+            setSuccessMessage(`Boosted ${data.amount} sats! ⚡️`);
+          }
+          return;
+        } catch (err) {
+          // user cancelled WebLN
+          // proceed with regular QR code
+        }
+      }
+
+      setPaymentRequest(paymentRequest);
+      setIsLoading(false);
+
+      poll({
+        fn: checkInvoice,
+        data: { paymentHash: paymentHash },
+        interval: 12000,
+        maxAttempts: 12,
+      })
+        .then(() => {
+          // success
+          setSuccessMessage(`Boosted ${data.amount} sats! ⚡️`);
+        })
+        .catch(() => {
+          // failure
+          console.error("Boost check failed");
+        });
+    } catch (e) {
+      console.error(e);
+    }
+  };
+
+  const methods = useForm({
+    defaultValues: {
+      amount: undefined,
+      comment: undefined,
+    },
+  });
+
+  const resetBoostPage = () => {
+    methods.reset();
+    setPaymentRequest("");
+    setSuccessMessage("");
+  };
+
+  if (successMessage) {
+    resetBoostPage();
+
+    setTimeout(() => {
+      backToPlayer();
+    }, 4000);
+
+    return (
+      <div className="flex h-full w-full flex-col justify-center rounded-3xl bg-brand-black p-4">
+        <p className="text-lg text-white">{successMessage}</p>
+        <button
+          onClick={backToPlayer}
+          type="button"
+          className="rounded-xl bg-brand-pink px-4 py-2 text-white"
+        >
+          Back
+        </button>
+      </div>
+    );
+  }
+
+  if (isLoading) {
+    return (
+      <div className="flex h-full w-full flex-col items-center justify-center rounded-3xl bg-brand-black p-4">
+        <p className="text-lg text-white">Processing...</p>
+      </div>
+    );
+  }
+
+  return (
+    <div className="flex h-full w-full flex-col justify-center rounded-3xl bg-brand-black p-4">
+      <button
+        onClick={backToPlayer}
+        type="button"
+        className="self-end py-2 pr-3"
+      >
+        Back
+      </button>
+      {paymentRequest ? (
+        <PaymentScreen
+          paymentRequest={paymentRequest}
+          setPaymentRequest={setPaymentRequest}
+        />
+      ) : (
+        <FormProvider {...methods}>
+          <form
+            className="flex flex-grow flex-col items-center gap-4"
+            onSubmit={methods.handleSubmit(onSubmit)}
+          >
+            <TextInput
+              fieldName="amount"
+              type="number"
+              title="Amount (sats)"
+              helperText="Min 1 sat, max 100,000 sats"
+              options={{
+                required: "Please enter an amount",
+                min: {
+                  value: 1,
+                  message: "Must zap at least something, like 1 sat",
+                },
```

Comment:
Let's bump up the min
```suggestion
              helperText="Min 10 sats, max 100,000 sats"
              options={{
                required: "Please enter an amount",
                min: {
                  value: 10,
                  message: "Must zap at least something, like 10 sats",
                },
```